### PR TITLE
Add setPaletteColor and rgb8 to help files

### PR DESCRIPTION
**Pull request by ObloxCC**
Title says it all

### CODE REVIEW

File: src/main/resources/assets/computercraft/lua/rom/help/colors [hunk 1]
```diff
@@ -3,6 +3,7 @@ Functions in the colors api
 colors.combine( color1, color2, color3, ... )
 colors.subtract( colors, color1, color2, ... )
 colors.test( colors, color )
+colors.rgb8( r , g , b)
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/colors]: The proper formatting is `colours.rgb8( r, g, b )`.

File: src/main/resources/assets/computercraft/lua/rom/help/colours [hunk 2]
```diff
@@ -2,7 +2,8 @@ Functions in the colours api
 (used for redstone.setBundledOutput):
 colours.combine( colour1, colour2, colour3, ...)
 colours.subtract( colours, colour1, colour2, ...)
+colours.rgb8( r , g , b )
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/colours]: Ditto

File: src/main/resources/assets/computercraft/lua/rom/help/term [hunk 3]
```diff
@@ -15,6 +15,7 @@ term.getSize()
 term.scroll( n )
 term.redirect( object )
 term.current()
+term.setPaletteColor( color , r , g , b)
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/term]: `term.setPaletteColor( color, r, g, b )`

**Comment by dan200** [comment]
Also monitor needs updating. And whagsnew and changelog

File: src/main/resources/assets/computercraft/lua/rom/help/window [hunk 4]
```diff
@@ -21,3 +21,4 @@ redraw()
 restoreCursor()
 getPosition()
 reposition( x, y, width, height )
+setPaletteColor( color , r , g , b)
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/window]: `setPaletteColor( color, r, g, b )

File: src/main/resources/assets/computercraft/lua/rom/help/colors [hunk 5]
```diff
@@ -3,6 +3,7 @@ Functions in the colors api
 colors.combine( color1, color2, color3, ... )
 colors.subtract( colors, color1, color2, ... )
 colors.test( colors, color )
+colors.rgb8( r, g, b)
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/colors]: You're still missing a space at the end here...

File: src/main/resources/assets/computercraft/lua/rom/help/term [hunk 6]
```diff
@@ -15,6 +15,7 @@ term.getSize()
 term.scroll( n )
 term.redirect( object )
 term.current()
+term.setPaletteColor( color, r, g, b)
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/term]: ... and here

File: src/main/resources/assets/computercraft/lua/rom/help/window [hunk 7]
```diff
@@ -21,3 +21,4 @@ redraw()
 restoreCursor()
 getPosition()
 reposition( x, y, width, height )
+setPaletteColor( color, r, g, b)
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/window]: ... and also here.

File: src/main/resources/assets/computercraft/lua/rom/help/changelog [hunk 8]
```diff
@@ -8,6 +8,8 @@ New Features in ComputerCraft 1.80:
 * Monitor text now glows in the dark.
 * Added a "Pocket Computer upgrde API" so mod developers can add their own pocket upgrades.
 * Added pocket.equipBack()/pocket.unequipBack() to add/remove pocket upgrades.
+* Added term.setColorPalette() to change colors
```
Comment (Lignum) [inline src/main/resources/assets/computercraft/lua/rom/help/changelog]: This is called `term.setPaletteColor`, not `term.setColorPalette`.

**Comment by ObloxCC** [comment]
What was added to monitor?

**Comment by Lignum** [comment]
The same functions as with term, `setPaletteColour`, `getPaletteColour`.

**Comment by dan200** [comment]
The same methods as we're added to term

File: src/main/resources/assets/computercraft/lua/rom/help/window (hunk 7 above)
Comment (dan200) [inline src/main/resources/assets/computercraft/lua/rom/help/window]: Also missing getPaletteColor

File: src/main/resources/assets/computercraft/lua/rom/help/term (hunk 6 above)
Comment (dan200) [inline src/main/resources/assets/computercraft/lua/rom/help/term]: Missing getPaletteColor

File: src/main/resources/assets/computercraft/lua/rom/help/monitors [hunk 9]
```diff
@@ -15,7 +15,8 @@ getTextColor()
 getBackgroundColor()
 getSize()
 scroll( n )
+setPalatteColor( color, r, g, b )
```
Comment (dan200) [inline src/main/resources/assets/computercraft/lua/rom/help/monitors]: It's Palette, not Palatte.
Also getPaletteColor is missing

**Comment by dan200** [comment]
Merged. Seconding please use GitHub Desktop, it's lovely.

**Comment by apemanzilla** [comment]
Or better yet, use `git` from the command line 😄

**Pull request merged**